Rx.playground/Pages/Playground.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,22 @@
+/*:
+ > # IMPORTANT: To use **Rx.playground**:
+ 1. Open **Rx.xcworkspace**.
+ 1. Build the **RxSwift-macOS** scheme (**Product** → **Build**).
+ 1. Open **Rx** playground in the **Project navigator**.
+ 1. Show the Debug Area (**View** → **Debug Area** → **Show Debug Area**).
+ */
+import RxSwift
+/*:
+ # Try Yourself
+ 
+ It's time to play with Rx 🎉
+ */
+playgroundShouldContinueIndefinitely()
+
+example("Try yourself") {
+  let disposeBag = DisposeBag()
```

Comment:
The reason why I'm suggesting that is because this is intended for playing around, and you've already prepared the codebase with `playgroundShouldContinueIndefinitely` for playing with time based operators.

That means that if some new user replaces your code with
```
let disposeBag = DisposeBag()
_ = Observable<Int>.interval(0.2, scheduler: MainScheduler.instance)
  .debug("Observable")
  .subscribe()
  .disposed(by: disposeBag)
```

we'll get a bug report in best case, or user will be confused and just give up figuring out what has gone wrong.